project/dependencies.scala

```diff
@@ -46,6 +46,11 @@ object Dependencies {
     "com.crawljax" % "crawljax-core" % "3.6",
     "com.crawljax.plugins" % "crawloverview-plugin" % "3.6",
     "com.crawljax" % "crawljax-cli" % "3.6",
+    "org.apache.kafka" % "kafka_2.10" % "0.10.0.0"
+      exclude("javax.jms", "jms")
```

Comment:
Because the package slf4j is also being included by twitter4J, so I am getting "multiple bindings" warns if I don't exclude it.
However, the others doesn't seem to cause harm. I will keep them.